CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ Current
 
 ### Added:
 
+
+- [dateTime based sort feature for the final ResultSet added](https://github.com/yahoo/fili/pull/178)
+   * Now we can sort the final result set in DESC order. Druid provided results are in ASC order by default.
+   * Added `DateTimeSortMapper` to reverse the time buckets and `DateTimeSortRequestHandler` to inject to the workflow
```

Comment:
Might want to generalize this to say that we not support sorting time in ascending or descending order.